### DISCUSSION TITLE
所以，，他到底要如何才能工作？

### DISCUSSION THREAD

**Discussion by cancundeyingzi** [Mygod/VPNHotspot#301] [Q&A]
研究了几天了，，各种按钮都按了，，一点效果都没。。。。

**Comment by Mygod** [comment]
This might help: https://blog.csdn.net/wang805447391/article/details/81772210

**Reply by cancundeyingzi** [reply]
感谢，dhcp的问题，，不过他好像依旧无法让仅对单个app生效的vpn起作用，，（以选择对该软件生效
